packages/react/src/contexts/treasure.tsx

```diff
@@ -55,6 +56,7 @@ type Config = {
   sessionOptions?: SessionOptions;
   autoConnectTimeout?: number;
   onConnect?: (user: User) => void;
+  getAuthTokenOverride?: () => string | undefined;
```

Comment:
Thoughts on nesting this into an object like `launcherOptions`? I just think in its current form it's unclear that this is for the launcher only